integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,243 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.0.1
+  namespace: placeholder
+  annotations:
+    alm-examples: "{}"
+    categories: Integreating greatly
+    certified: 'false'
+    containerImage: quay.io/integreatly/integreatly-operator:1.1.0
```

Comment:
has this image been pushed to this repo?

Reply:
yep